Formula/kubectx.rb

```diff
@@ -0,0 +1,26 @@
+class Kubectx < Formula
+  desc "Tool that can switch between kubectl contexts easily and create aliases"
+  homepage "https://github.com/ahmetb/kubectx"
+  url "https://github.com/ahmetb/kubectx/archive/v0.3.1.tar.gz"
+  sha256 "4e995f5bec6f41c8d5b6e77f413a58ead077816348e72de26dde3655ec2b7d0b"
+  head "https://github.com/ahmetb/kubectx.git", :branch => "master"
+  bottle :unneeded
+
+  option "with-short-names", "link as \"kctx\" and \"kns\" instead"
+
+  def install
+    bin.install "kubectx" => build.with?("short-names") ? "kctx" : "kubectx"
+    bin.install "kubens" => build.with?("short-names") ? "kns" : "kubens"
+    include.install "utils.bash"
+
+    bash_completion.install "completion/kubectx.bash" => "kubectx"
+    bash_completion.install "completion/kubens.bash" => "kubens"
+    zsh_completion.install "completion/kubectx.zsh" => "_kubectx"
+    zsh_completion.install "completion/kubens.zsh" => "_kubens"
+  end
+
+  test do
+    system "which", build.with?("short-names") ? "kctx" : "kubectx"
```

Comment:
they exit with code=0 when there's no ~/.kube. without a .kube, I am not sure how much better we can test. calling `--help` on these commands could be a basic smoke test that also exits with code=0 when there's no .kube, but it would test that the dependencies are loaded correctly.

```
$ kubectx; echo $?
0
$ kubens; echo $?
the server doesn't have a resource type "namespaces"
0
```